core/src/main/res/values/strings.xml

```diff
@@ -24,5 +24,8 @@
     <string name="device_view_title">Device Information</string>
     <string name="logs_view_title">Logs</string>
     <string name="file_issue_view_title">File Issue</string>
+    <string name="search_logs_hint">Search the Logs</string>
+    <string name="search_logs_button_text">Search</string>
+    <string name="placeholder_logs">Temporary text to be replaced by logs</string>
     <string name="placeholder_device_info">Temporary text to be replaced by device info</string>
```

Comment:
```suggestion
    <string name="placeholder_device_info">Waiting for device info…</string>
```

Reply:
The placeholder_device_info string resource is removed in the PR to show the actual device information on the developer menu (https://github.com/aws-amplify/amplify-android/pull/660).